common/crypto_utils/avalon_crypto_utils/crypto_utility_using_avalon_crypto_functions.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2020 Intel Corporation
```

Comment:
New Avalon crypto files are not required since the goal of this PR is to replace existing Avalon crypto functions